servers/src/main/java/tachyon/master/file/FileSystemMaster.java

```diff
@@ -1783,6 +1806,166 @@ private void setStateFromEntry(SetStateEntry entry) throws FileDoesNotExistExcep
   }
 
   /**
+   * Sets the acl of a file or directory.
+   *
+   * @param path to be set acl on
+   * @param options acl option to be set
+   * @return true if set successfully, false otherwise
+   * @throws TachyonException
+   */
+  public boolean setAcl(TachyonURI path, SetAclOptions options) throws TachyonException {
+    if (options.getOwner() != null) {
+      return setOwner(path, options.getOwner(), options.isRecursive());
```

Comment:
Yes, the intention is that make it consistent with the behavior at client side. The commands `chown`, `chgrp`, `chmod` can only set one of them in a call.

Added a precondition check to ensure only one of them take effect in a call.